src/__tests__/App.test.js

```diff
@@ -1,3 +1,11 @@
 import { render, screen } from '@testing-library/react';
 import App from '../components/App';
 
+describe("app", () => {
```

Comment:
None of your tests are running. This is because of the version of axios youre using, they should fix it at some point but until then try adding this to your package.json:

```
"jest": {
  "transformIgnorePatterns": ["node_modules/(?!axios)/"]
}
```

Or update the “test” line in your package.json to this:

```
"test": "react-scripts test --transformIgnorePatterns \"node_modules/(?!axios)/\""
```